websites/M/MultiplayerPiano.net/metadata.json

```diff
@@ -0,0 +1,36 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.9",
+	"author": {
+		"id": "224864947480297472",
+		"name": "antokek"
+	},
+	"service": "MultiplayerPiano.net",
+	"description": {
+		"en": "An online website used for playing piano with friends. It supports input from both MIDI keyboards and QWERTY keyboards."
+	},
+	"url": "multiplayerpiano.net",
+	"version": "1.0.0",
+	"logo": "https://cdn.discordapp.com/icons/813217128529264710/911b06d8a1a9af24fac16b843165cc0f.png",
```

Comment:
The [logo](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md#logo) should be a 512x512px image